ee/modules/650-runtime-audit-engine/docs/FAQ_RU.md

```diff
@@ -0,0 +1,103 @@
+---
+title: "Модуль runtime-audit-engine: примеры конфигурации"
+---
+
+## Как собирать события?
+
+Pod'ы `runtime-audit-engine` выводят все события в стандартный вывод.
+Далее [агенты log-shipper](../460-log-shipper/) могут собирать их и отправлять в хранилище логов.
+
+Пример конфигурации [ClusterLoggingConfig](/460-log-shipper/cr.html#clusterloggingconfig) для модуля `log-shipper`:
+
+```yaml
+apiVersion: deckhouse.io/v1alpha1
+kind: ClusterLoggingConfig
+metadata:
+  name: falco-events
+spec:
+  destinationRefs:
+  - xxxx
+  kubernetesPods:
+    namespaceSelector:
+      matchNames:
+      - d8-runtime-audit-events
+  labelsFilter:
+  - operator: Regex
+    values: ["\{.*"] # to collect only JSON logs
+    field: "message"
+  type: KubernetesPods
+```
+
+## Как оповещать о критических событиях?
+
+Prometheus автоматически собирает метрики о событиях.
+Чтобы включить оповещения, добавьте в кластер правило [CustomPrometheusRule](../300-prometheus/cr.html#customprometheusrules).
+
+Пример настройки такого правила:
+
+```yaml
+apiVersion: deckhouse.io/v1
+kind: CustomPrometheusRules
+metadata:
+  name: falco-critical-alerts
+spec:
+  groups:
+  - name: falco-critical-alerts
+    rules:
+    - alert: FalcoCriticalAlertsAreFiring
+      for: 1m
+      annotations:
+        description: |
+          There is a suspicious activity on a node {{ $labels.node }}. 
+          Check you events journal for more details.
+        summary: Falco detects a critical security incident
+      expr: |
+        sum by (node) (rate(falco_events{priority="Critical"}[5m]) > 0)
+```
+
+> NOTE: Алерты лучше всего работают в комбинации с хранилищами событий, такими как Elasticsearch или Loki. Их задача — оповестить пользователя о подозрительном поведении на узле.
```

Comment:
```suggestion
> Алерты лучше всего работают в комбинации с хранилищами событий, такими как Elasticsearch или Loki. Их задача — оповестить пользователя о подозрительном поведении на узле.
```

IMHO If this is just additional information, not a warning, then `NOTE:` is excessive here. If it is important info, then let's use **Note, that**  or Obratytte vnymanie, chto...